src/commcare_cloud/ansible/roles/setup_auth_keys.yml

```diff
@@ -18,13 +18,13 @@
     fail_on_missing: yes
   ignore_errors: "{{ ansible_check_mode }}"
 
-- name: "{{ 'Add' if state == 'present' else 'Remove' }} public key on {{ hostTo }}"
+- name: "{{ 'Add' if state == 'present' else 'Remove' }} public key on {{ hostTo }} for {{ userTo }}"
   become: true
   authorized_key:
-    user: "{{username}}"
+    user: "{{userTo}}"
     state: "{{ state }}"
     key: "{{ lookup('file', '/tmp/id_rsa.tmp') }}"
-  delegate_to: "{{ inventory_hostname }}"
+  delegate_to: "{{ hostTo }}"
   ignore_errors: "{{ ansible_check_mode }}"
 
 - name: Deleting temp files
```

Comment:
This file is also referenced here: https://github.com/dimagi/commcare-cloud/blob/e0e133bfa4263d168659b494975525d6cda545e9/src/commcare_cloud/ansible/deploy_db.yml#L39